src/donor_dashboard/templates/donor_dashboard/manage_organization.html

```diff
@@ -14,17 +14,21 @@ <h2 class="subtitle mt-5">
             </h2>
             <div id="organization-id" hidden>{{ organization.organization_id }}</div>
                 {% if status %}
-                    <div class="buttons">
-                        <button id="add-donation-button" class="button is-dark">Add Donation</button>
-                        <a href="{% url 'donor_dashboard:organization_details' organization.organization_id %}" class="button is-info">
-                            Organization Details
-                        </a>
+                    
+                        <div class="buttons">
+                            <button id="add-donation-button" class="button is-dark">Add Donation</button>
+                            {% if owner_access %}
+                            <a href="{% url 'donor_dashboard:organization_details' organization.organization_id %}" class="button is-info">
+                                Organization Details
```

Comment:
Nit: Since we are managing the organization here, should we name it "Manage Organization" instead of organization details so that it is self explanatory? We would also have to update the button name in the list of organizations to "View Organization" to avoid confusion

Reply:
That makes sense. Will raise a new PR for this.

Reply:
Very minor stuff, so made changes in this PR itself.